osx/FontForge.app/Contents/Info.plist.in

```diff
@@ -1,409 +1,410 @@
+				</array>
+				<key>LSTypeIsPackage</key>
+				<false />
+			</dict>
+			<dict>
+				<key>CFBundleTypeExtensions</key>
+				<array>
+					<string>otf</string>
+				</array>
+				<key>CFBundleTypeIconFile</key>
+				<string>fontforge_otf</string>
+				<key>CFBundleTypeName</key>
+				<string>OpenType</string>
+				<key>CFBundleTypeOSTypes</key>
+				<array>
+					<string>OTTO</string>
+				</array>
+				<key>CFBundleTypeRole</key>
+				<string>Viewer</string>
+				<key>LSItemContentTypes</key>
+				<array>
+					<string>public.opentype-font</string>
+				</array>
+				<key>LSTypeIsPackage</key>
+				<false />
+			</dict>
+			<dict>
+				<key>CFBundleTypeIconFile</key>
+				<string>fontforge_ps</string>
+				<key>CFBundleTypeName</key>
+				<string>Mac PostScript outline</string>
+				<key>CFBundleTypeOSTypes</key>
+				<array>
+					<string>LWFN</string>
+				</array>
+				<key>CFBundleTypeRole</key>
+				<string>Viewer</string>
+				<key>LSItemContentTypes</key>
+				<array>
+					<string>com.adobe.postscript-lwfn-font</string>
+				</array>
+				<key>LSTypeIsPackage</key>
+				<false />
+			</dict>
+			<dict>
+				<key>CFBundleTypeExtensions</key>
+				<array>
+					<string>pfa</string>
+					<string>pfb</string>
+				</array>
+				<key>CFBundleTypeIconFile</key>
+				<string>fontforge_ps</string>
+				<key>CFBundleTypeName</key>
+				<string>PostScript outline</string>
+				<key>CFBundleTypeRole</key>
+				<string>Viewer</string>
+				<key>LSItemContentTypes</key>
+				<array>
+					<string>com.adobe.postscript-pfb-font</string>
+					<string>com.adobe.postscript-pfa-font</string>
+				</array>
+				<key>LSTypeIsPackage</key>
+				<false />
+			</dict>
+			<dict>
+				<key>CFBundleTypeExtensions</key>
+				<array>
+					<string>suit</string>
+				</array>
+				<key>CFBundleTypeIconFile</key>
+				<string>fontforge_sfd</string>
+				<key>CFBundleTypeName</key>
+				<string>Font Suitcase</string>
+				<key>CFBundleTypeOSTypes</key>
+				<array>
+					<string>FFIL</string>
+					<string>ffil</string>
+					<string>sfnt</string>
+					<string>tfil</string>
+				</array>
+				<key>CFBundleTypeRole</key>
+				<string>Viewer</string>
+				<key>LSItemContentTypes</key>
+				<array>
+					<string>com.apple.font-suitcase</string>
+				</array>
+				<key>LSTypeIsPackage</key>
+				<false />
+			</dict>
+			<dict>
+				<key>CFBundleTypeExtensions</key>
+				<array>
+					<string>ttf</string>
+				</array>
+				<key>CFBundleTypeIconFile</key>
+				<string>fontforge_ttf</string>
+				<key>CFBundleTypeName</key>
+				<string>TrueType Font</string>
+				<key>CFBundleTypeRole</key>
+				<string>Viewer</string>
+				<key>LSItemContentTypes</key>
+				<array>
+					<string>public.truetype-ttf-font</string>
+				</array>
+				<key>LSTypeIsPackage</key>
+				<false />
+			</dict>
+		</array>
+		<key>CFBundleName</key>
+		<string>FontForge</string>
+		<key>CFBundleExecutable</key>
+		<string>../Resources/opt/local/bin/fontforge</string>
```

Comment:
As mentioned in that issue thread, the bundle now executes the binary directly instead of going through a script.

This will cause Gatekeeper to prompt the user if they wish to allow FF to access that location instead of silently denying access.